LightBlue.MultiHost/Runners/DotNetCoreRunner.cs

```diff
@@ -30,7 +30,7 @@ public void Start()
             _parent = new Process
             {
                 EnableRaisingEvents = true,
-                StartInfo = new ProcessStartInfo("dotnet", "run")
+                StartInfo = new ProcessStartInfo("dotnet", "run --no-build --no-restore")
```

Comment:
This would otherwise fail when a service has a reference to a DLL in use by another service (preventing the build from updating it).